internal/handlers/device.go

```diff
@@ -298,8 +305,11 @@ func (api *API) CreateDevice(c *gin.Context) {
 		}
 		// allocate a child prefix if requested
 		for _, prefix := range request.ChildPrefix {
```

Comment:
As far as I can tell, nothing is validating that the child prefixes are valid. They could be garbage and we're relying on `api.ipam.AssignPrefix()` to return an error. I don't think the code should make it that far if the input is obviously malformed.